source/features/table-input.tsx

```diff
@@ -37,11 +37,13 @@ function init(): void {
 
 	for (const anchor of select.all('md-task-list')) {
 		anchor.after(
-			<details className="details-reset details-overlay flex-auto toolbar-item select-menu select-menu-modal-right hx_rsm">
+			<details
+				className="toolbar-item tooltipped tooltipped-n mx-1 details-reset details-overlay flex-auto select-menu select-menu-modal-right hx_rsm"
+				aria-label="Add a table"
+			>
 				<summary
-					className="text-center menu-target py-2 p-md-1 hx_rsm-trigger ml-1"
+					className="text-center menu-target py-2 p-md-1 hx_rsm-trigger"
 					role="button"
-					aria-label="Add a table"
```

Comment:
The summary is the button, and the button needs the tooltip. Details is just the container

Reply:
This is what I mean by *the popup arrow being cut off*:

![chrome_9kVBeMxQ7l](https://user-images.githubusercontent.com/44045911/94990259-3f097e80-05ad-11eb-9599-f134be1380b3.png)

It is fine if the user just hovers on it (doesn't activate it) though. This is a minor issue, so I will leave it as is.